airflow/operators/python.py

```diff
@@ -131,14 +131,14 @@ def my_python_callable(**kwargs):
     :type show_return_value_in_logs: bool
     """
 
-    template_fields = ('templates_dict', 'op_args', 'op_kwargs')
+    template_fields: Sequence[str] = ('templates_dict', 'op_args', 'op_kwargs')
```

Comment:
Do we need to specify the typing here? Doesn't it automatically take the typing from the parent class?

Reply:
Unfortunately the typing doesn't inherit from `BaseOperator` here for some reason. With the typing removed we start seeing this:
```
airflow/decorators/python_virtualenv.py:45: error: Incompatible types in assignment (expression has type "Tuple[str, str]", base class "PythonOperator" defined the type as "Tuple[str, str, str]")
        template_fields = ('op_args', 'op_kwargs')
                           ^
airflow/decorators/python.py:42: error: Incompatible types in assignment (expression has type "Tuple[str, str]", base class "PythonOperator" defined the type as "Tuple[str, str, str]")
        template_fields = ('op_args', 'op_kwargs')
```

Reply:
Yeah, if you make the type on BaseOperator a Sequence instead of a Tuple it should fix this problem too, and then we shouldn't need the typing here.

Reply:
That's part of the updates I made as well.  `BaseOperator` is Sequence-typed here but it's not picked up in `PythonOperator`.

Reply:
Oh well :)